examples/block-explorer/explorer-index/src/lib.rs

```diff
@@ -75,42 +75,50 @@ mod explorer_index {
             let mut tx_amount = 0;
             let mut tokens_transferred = Vec::new();
 
-            // `Transaction::Script` and `Transaction::Create` are unused but demonstrate properties like gas, inputs,
-            // outputs, script_data, and other pieces of metadata.
+            // `Transaction::Script`, `Transaction::Create`, and `Transaction::Mint`
+            // are unused but demonstrate properties like gas, inputs,
+            // outputs, script_data, and other pieces of metadata. You can access
+            // properties that have the corresponding transaction `Field` traits
+            // implemented; examples below.
             match &tx.transaction {
                 #[allow(unused)]
-                Transaction::Script {
-                    gas_price,
-                    gas_limit,
-                    maturity,
-                    receipts_root,
-                    script,
-                    script_data,
-                    inputs,
-                    outputs,
-                    witnesses,
-                    metadata,
-                } => {
+                Transaction::Script(t) => {
+                    let gas_limit = t.gas_limit();
+                    let gas_price = t.gas_price();
+                    let maturity = t.maturity();
+                    let script = t.script();
+                    let script_data = t.script_data();
+                    let receipts_root = t.receipts_root();
+                    let inputs = t.inputs();
+                    let outputs = t.outputs();
+                    let witnesses = t.witnesses();
+
+                    let json = &tx.transaction.to_json();
                     Logger::info("Inside a script transaction. (>^‿^)>");
```

Comment:
Nit: This log should go before all of the variables (if you look at the diff you see the log came as soon as we were in the transaction before we did anything else)